oms/src/main/java/com/gfg/oms/response/Error.java

```diff
@@ -0,0 +1,20 @@
+package com.gfg.oms.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+@AllArgsConstructor
+@Data
+@NoArgsConstructor
+public class Error implements Serializable {
+    @JsonProperty("error")
+    private ErrorData error;
```

Comment:
In jackson there is no way to map directly `{"error": {"key": "value"}}` to an object without having property `error`. There are ways of manual parsing json tree, but I would like to keep all automatic mapping features